action/protocol/staking/protocol.go

```diff
@@ -267,8 +270,17 @@ func (p *Protocol) handle(ctx context.Context, act action.Action, csm CandidateS
 		logs = append(logs, l)
 	}
 	if err == nil {
-		if aLog != nil {
-			logs = append(logs, aLog)
+		if createLog != nil {
```

Comment:
maybe just use aLog?

Reply:
we now have 4 possible logs, so name them explicitly

Reply:
I think they are the same, for this function "handle", it doesn't need to understand what's the meaning of the logs. We may further unify handleXXX return values to (*receiptLog, []*action.Log, error)

Reply:
handleRegister() will return 2 extra logs, handleCreate/Deposit/Withraw() 1 extra log, others no extra log

would leave it as is, feel unifying to (*receiptLog, []*action.Log, error) does not make the program better/clear

Reply:
we have different understanding of programming